happi-graph.js

```diff
@@ -360,6 +364,9 @@ class HappiGraph extends PolymerElement {
 
   initGraph() {
     this.svg = d3.select(this.$.svg);
+    this.addEventListener('mousemove', function (event) {
+      this.mousePosition = {x: event.x, y: event.y};
+    })
```

Comment:
this needs to ne removed when component is removed from DOM, otherwise we'll have memory leaks and infinite unused eventlisteners

you can declare `this.mousePositionEventListener` with this and in `disconnectedCallback() {}` you should removeEventListener(this.MousePositionEventListener)